f2/src/locales/en.json

```diff
@@ -134,7 +134,6 @@
   "confirmationPage.whoAreYouReportFor.nag": "No information provided.",
   "confirmationPage.whoAreYouReportFor.title.edit": "Edit Who are you report for?",
   "confirmationPage.youAreReportingFor": "You are reporting for:",
```

Comment:
This message string is used in the analyst email. We may have to replace the ugly _mark ids for lingui_ section.